server/app/services/question/types/AddressQuestionDefinition.java

```diff
@@ -72,16 +72,24 @@ public static AddressValidationPredicates parse(String jsonString) {
     }
 
     public static AddressValidationPredicates create() {
-      return builder().setDisallowPoBox(false).build();
+      return builder().setDisallowPoBox(false).setDefaultState(Optional.empty()).build();
     }
 
     public static AddressValidationPredicates create(boolean disallowPoBox) {
-      return builder().setDisallowPoBox(disallowPoBox).build();
+      return builder().setDisallowPoBox(disallowPoBox).setDefaultState(Optional.empty()).build();
+    }
+
+    public static AddressValidationPredicates create(
+        boolean disallowPoBox, Optional<String> defaultState) {
+      return builder().setDisallowPoBox(disallowPoBox).setDefaultState(defaultState).build();
     }
 
     @JsonProperty("disallowPoBox")
     public abstract Optional<Boolean> disallowPoBox();
 
+    @JsonProperty("defaultState")
+    public abstract Optional<String> defaultState();
```

Comment:
Comment for @bion `defaultState` doesn't feel like it should be part of `AddressValidationPredicates` as it's not a validation. Should this field be set as part of different class?

Reply:
Agreed, the default state is unrelated to validation predicates and should not be stored here.

Reply:
Not quite, when fields are added on QuestionDefinition, Question needs to be updated to know how to persist them: https://github.com/civiform/civiform/blob/main/server/app/models/Question.java#L241

Rather than adding a field that's specific to one question type (`AddressQuestionDefinition`, in this case), a more general approach could be to add a new inner class on `QuestionDefinition` similar to `ValidationPredicates` that also serializes to JSON, and then inherit from it in an inner class in `AddressQuestionDefinition`, similar to how `AddressValidationPredicates` inherits from `ValidationPredicates`.

Reply:
@bion -  i can do it but it will increase the size and scope of PR a lot. I will be again stuck with a huge PR and unable to checkin. Is there a way to break it up?